soc/raspberrypi/rpi_pico/common/pinctrl_soc.h

```diff
@@ -42,17 +48,20 @@ typedef struct rpi_pinctrl_soc_pin pinctrl_soc_pin_t;
  * @param prop Property name.
  * @param idx Property entry index.
  */
-#define Z_PINCTRL_STATE_PIN_INIT(node_id, prop, idx)				\
-	{									\
-		RP2_GET_PIN_NUM(DT_PROP_BY_IDX(node_id, prop, idx)),		\
-		RP2_GET_PIN_ALT_FUNC(DT_PROP_BY_IDX(node_id, prop, idx)),	\
-		DT_ENUM_IDX(node_id, drive_strength),				\
-		DT_ENUM_IDX(node_id, slew_rate),				\
-		DT_PROP(node_id, bias_pull_up),					\
-		DT_PROP(node_id, bias_pull_down),				\
-		DT_PROP(node_id, input_enable),					\
-		DT_PROP(node_id, input_schmitt_enable),				\
-		DT_PROP(node_id, raspberrypi_oe_override),				\
+#define Z_PINCTRL_STATE_PIN_INIT(node_id, prop, idx)                                               \
+	{                                                                                          \
+		RP2_GET_PIN_NUM(DT_PROP_BY_IDX(node_id, prop, idx)),                               \
+		RP2_GET_PIN_ALT_FUNC(DT_PROP_BY_IDX(node_id, prop, idx)),                          \
+		DT_ENUM_IDX(node_id, drive_strength),                                              \
+		DT_ENUM_IDX(node_id, slew_rate),                                                   \
+		DT_PROP(node_id, bias_pull_up),                                                    \
+		DT_PROP(node_id, bias_pull_down),                                                  \
+		DT_PROP(node_id, input_enable),                                                    \
+		DT_PROP(node_id, input_schmitt_enable),                                            \
+		DT_PROP(node_id, raspberrypi_oe_override),                                         \
+		DT_PROP(node_id, raspberrypi_out_override),                                        \
+		DT_PROP(node_id, raspberrypi_in_override),                                         \
```

Comment:
```suggestion
		DT_PROP(node_id, raspberrypi_output_override),                                        \
		DT_PROP(node_id, raspberrypi_input_override),                                         \
```
In combination with the suggestion above.